contract/models/abstract_contract.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2004-2010 OpenERP SA
+# Copyright 2014 Angel Moya <angel.moya@domatix.com>
+# Copyright 2015 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2016-2018 Carlos Dauden <carlos.dauden@tecnativa.com>
+# Copyright 2016-2017 LasLabs Inc.
+# Copyright 2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models, fields
+
+
+class AbstractAccountAnalyticContract(models.AbstractModel):
```

Comment:
Why the incredibly long name for class and model? I would prefer just AbstractContract and 'abstract.contract' (and the same goes for line). This would also give full conistency between python filename, model name and class name.

Reply:
Agree,

Reply:
@sbidoul summarized why we are in this "just between" situation